### PR TITLE
web4975 - addition of distance between component

### PR DESCRIPTION
**Pull request by nickine**
#### What does this PR do?
renders distance between 2 sets of coordinates and point of interest
#### What unit or integration tests does this PR have?
N/A
#### What selenium tests does this PR have?
N/A
#### How should a developer review this?
Check the code looks sane and follow the manual testing steps below
#### How should this be manually tested?
One you have checked out the branch
npm install
npm run build-dev
and open dev/index.html to view the component
#### Any background context you want to provide?
This is part of making components for all the things
#### What are the relevant tickets?
https://hxshortbreaks.atlassian.net/browse/WEB-4975
#### Screenshots (if appropriate)
N/A
#### What gif best describes how you feel about this work?
vst29at

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've checked there is appropriate selenium test coverage.
- [x] I've included regression tests for any bug fixes.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this on mobile Android, Safari, Chrome for any visual changes.
- [x] I've tested this on desktop IE9/10/11, Firefox, Chrome for any visual changes.
- [x] I've checked any significant visual changes with a member of the UX/UI pod.
- [x] I've checked this work against the requirements of the Jira.
- [x] I've added and updated translations for all supported languages
- [x] I've informed those who need to know of any unforeseen impact of this work (database migrations, deployment procedure changes).
- [x] I am ready for this to be code reviewed, merged and tested.
- [x] I've thought about writing a blog post about this

#### IP or Developer Review:
- [ ] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [ ] I’ve run all the tests locally and they pass.
- [ ] +1 from me!

#### Software Engineer or Developer review:
- [ ] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [ ] I’ve run all the tests locally and they pass.
- [ ] I’ve checked for appropriate test coverage.
- [ ] I’ve checked for coding anti-patterns.
- [ ] I've checked this work against the requirements of the Jira.
- [ ] +1 from me!

#### Software Engineer or project guru review:
- [ ] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [ ] I’ve run all the tests locally and they pass.
- [ ] I’ve checked for appropriate test coverage.
- [ ] I’ve checked for coding anti-patterns.
- [ ] I don’t believe this work introduces unnecessary technical debt.
- [ ] +1 from me!

### CODE REVIEW

**Comment by lukehansell-hx** [comment]
taking SE/Guru

File: code/templates/distance_between_component_template.jsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 var React = require('react');
 
-module.exports = function(props) {
+module.exports = function(props, distanceCalculator) {
   return (
-    <div className="ui-component-distance-between">
-    {/* Insert your component html here */}
-    </div>
+    <p className="ui-component-distance-between distance">
+      <span>{distanceCalculator( props.pointA, props.pointB, props.decimalPlaces, props.inMiles )}</span> {props.inMiles ? 'miles' : 'km'} from {props.place}
```
Comment (lukehansell-hx) [inline code/templates/distance_between_component_template.jsx]: I think we need to have a discussion as to how to make this multi-lingual.
We need to be considerate of languages in which word order may change as well as the words themselves.
We've also again got the issue of html in the translations...

**Comment by lukehansell-hx** [comment]
@michaelcarter raised an interesting point. Perhaps this should actually be a shared helper function (an npm module) to do the calculation. This component doesn't actually return anything more than a span with text in. That way we can reuse this functionality elsewhere (even where we're not displaying html).

**Comment by lukehansell-hx** [comment]
@michaelcarter raised an interesting point. Perhaps this should actually be a shared helper function (an npm module) to do the calculation. This component doesn't actually return anything more than a span with text in. That way we can reuse this functionality elsewhere (even where we're not displaying html).

**Comment by nickine** [comment]
Component is unnecessary as it can be solved with npm module.

**Pull request closed without merging**